frappe/hooks.py

```diff
@@ -88,6 +90,9 @@
 }
 
 doc_events = {
+	"User": {
+		"validate": "frappe.utils.user.validate_user_limit"
```

Comment:
We've moved code related to each limit type to the respective doctypes / util files. Eg : File limit functions are in file.py